docs/types/contexts.md

```diff
@@ -43,59 +47,78 @@ the compiler, and hence can be automatically lifted to aid usability.
 
 ## Context Syntax
 
-There are three main notes about the syntax of contexts:
-
-1. Monadic contexts are defined using the `in` keyword (e.g. `Int in IO`).
-2. We have a symbol `!`, which is short-hand for putting something into the
-   `Exception` monadic context. This is related to broken values.
-3. Contexts can be combined by using the standard typeset operators, or nested
-   through repeated uses of `in`.
+> [!WARNING]
+> There used to be three main notes about the syntax of contexts:
+>
+> 1. Monadic contexts are defined using the `in` keyword (e.g. `Int in IO`).
+> 2. We have a symbol `!`, which is short-hand for putting something into the
+>   `Exception` monadic context. This is related to broken values.
+> 3. Contexts can be combined by using the standard typeset operators, or nested
+>   through repeated uses of `in`.
+
+There is no special syntax for contexts anymore.
+Since [#3828](https://github.com/enso-org/enso/pull/3828) 
+Enso is no longer relaying on a haskelly solution. 
+Rather than that _contexts_ are being manupulated by
+_standard library_ functions grouped around 
+`Standard.Base.Runtime.Context` & co.
+```ruby
+Runtime.Context.Output.with_enabled <|
+    File.new "c:\trash.txt" . delete
+```
+
+There is still the `!` symbol signaling [presence of errors](./errors.md)
+- e.g. _broken values_. However the runtime can handle _broken values_ 
+even without presence of these _exception type signatures_. Thus the
+compiler only verifies the referenced types are valid.
 
 ## Monadic Bind
 
-It is also important to note that Enso has no equivalent to `<-` in Haskell.
-Instead, pure computations are implicitly placed in the `Pure` monadic context,
-and `=` acts to 'peel off' the outermost layer of contexts. As such, this means
-that `=` _always_ acts as `bind`, greatly simplifying how the type-checker has
-to work.
+> [!WARNING]
+> Who knows what `<-` means in Haskell?
+> 
+> It is also important to note that Enso has no equivalent to `<-` in Haskell.
+> Instead, pure computations are implicitly placed in the `Pure` monadic context,
+> and `=` acts to 'peel off' the outermost layer of contexts. As such, this means
+> that `=` _always_ acts as `bind`, greatly simplifying how the type-checker has
+> to work.
 
 ## Context Definitions
 
-Contexts can be defined by users.
-
-> The actionables for this section are:
->
-> - How, what, when and why?
+The supported contexts are `Input`, `Output` and `Dataflow_Stack_Trace` as of
+Enso 2024.5.1 version. Users cannot define their own.
 
 ## Context Lifting
 
-> The actionables for this section are:
+> [!WARNING]
+> Is there any meaning of _lifting_ when there is no special support in type system?
```

Comment:
I think it no longer is applicable if the 'Context' is just basically a state.

Reply:
Thank you, I am deleting this section in 5ec72a3d1e